### PR TITLE
Support organization package

### LINKED ISSUE #10: Support organization package
Current API doesn't support package with an organization namespace like `@nodesecure/`. In case of an error the page will load indefinitely without informing the user of an error.

image

I guess this is because the `/` should be escaped for `%2F` in `index.tsx`:

```ts
const res = await fetch(`/api/scanner/${pkgName.replace("/", "%2F")}`);
```

For example in the CLI:
- https://github.com/ES-Community/nsecure/blob/master/public/js/utils.js#L217
- https://github.com/ES-Community/nsecure/blob/master/src/http-server/bundle.js#L9

### PR DESCRIPTION
**Pull request by viterb-c**
Replace '/' by '%2F' in package name for api call and next router

Fix #10

### CODE REVIEW

**Review by tony-go — CHANGES_REQUESTED** [review]
Awesome 👏 

Could you add an e2e test for that?

You can refer to `README.md` to know how to install required dependencies and how to run them.

**Comment by viterb-c** [comment]
@tony-go Done :)

**Review by tony-go — APPROVED** [review]
LGTM 🚀

**Review by fraxken — APPROVED** [review]
LGTM

File: pages/index.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ const Home: NextPage = () => {
     if (maybeCached) {
       analysis = JSON.parse(maybeCached);
     } else {
-      const res = await fetch(`/api/scanner/${pkgName}`);
+      const res = await fetch(`/api/scanner/${pkgName.replace("/", "%2F")}`);
```
Comment (tony-go) [inline pages/index.tsx:34]: I thought about creating a helper function as we use the same twice. But it seems already really expressive. WDYT?

Comment (tony-go) [reply pages/index.tsx:34]: We could make it afterward anyway ^^

**Pull request merged**